styles/Canonical/020-Cliche-words-and-phrases.yml

```diff
@@ -0,0 +1,30 @@
+# 20 - Cliché words and phrases - avoid these.
+
+extends: existence
+message: "Avoid the phrase '%s'"
+link: https://docs.ubuntu.com/styleguide/en#words-and-phrases-to-avoid
+level: warning
+ignorecase: true
+tokens:
+  - 'allow.*?'
+  - ability to 
+  - able to 
+  - 'not only.*?but also'
+  - 'eliminat.*?'
+  - 'execute.*?'
+  - 'terminate.*?'
+  - 'kill.*?'
+  - 'disrupt.*?'
```

Comment:
Agreed - good suggestion!

I also assumed the behavioral context - i.e. to avoid markety clichés about a software product being "disruptive in the industry". 

For usages like the disruption of some technical service, I don't see why this should be in the list.